packages/pilot/src/containers/Balance/index.js

```diff
@@ -114,6 +114,8 @@ const getSelectedPreset = (end, start) => {
   return 'period'
 }
 
+const isPaymentLink = company => path(['type'], company) === 'payment_link_app'
```

Comment:
Pode usar [propEq](https://ramdajs.com/docs/#propEq) aqui:

```suggestion
const isPaymentLink = propEq('type', 'payment_link_app')
```